src/lib/viewers/doc/_docBase.scss

```diff
@@ -55,7 +55,6 @@ $thumbnail-sidebar-width: 201px;
         flex: 1 0 auto;
         overflow: hidden;
         padding: 0;
-        width: 134px;
```

Comment:
Was this not needed?

Reply:
Yeah I don't think so because the flexbox width seems to take precedence

Reply:
I take it back, ie11 needs this, but I'll set it to the flex-basis instead